packages/forklift-console-plugin/src/components/page/StandardPage.tsx

```diff
@@ -412,6 +419,17 @@ export function StandardPage<T>({
                   <Action key={index} dataOnScreen={showPagination ? pageData : filteredData} />
                 ))}
             </ToolbarToggleGroup>
+
+            {selectedCountLabel && (
+              <ToolbarItem
+                className="pf-v5-u-color-200"
+                align={{ default: 'alignRight' }}
+                alignSelf="center"
```

Comment:
u already using CN , so why not adding also a custom selector? - u will keep all ur stying in one place, currently its in CN and pros.

Reply:
I mean u have 2 ways to handle styling in this comp, 1 is CN and 2 us the props.
Which combine in the end for the styling.

It makes it harder to maintain the component styling when it is split like this. Consider only using CN, u can use @extand in your CN to add the .pf-v5-u-color-200 0 we are using scss

Reply:
I disagree that this is harder to maintain. All of the styles that apply to this component are currently co-located where the component is used and there is no need to create a new scss file or add lines somewhere else that will take discovery time to find and update. Also again, using everything that comes from patternfly vs custom styles leads to easier migration to newer versions of patternfly.

Reply:
Sent u a Slack msg :)